cmake/modules/Findkeyutils.cmake

```diff
@@ -13,7 +13,7 @@ INCLUDE(CheckCXXSymbolExists)
 
 # include dir
 
-find_path(KEYUTILS_INCLUDE_DIR keyutils.h NO_DEFAULT_PATH PATHS
+find_path(KEYUTILS_INCLUDE_DIR keyutils.h PATHS
```

Comment:
@tchaikov I just did the minimal change to get it working. But I will do the other cleanups as soon later today. Is there a chance to get this backportet to the kraken branch later?